2019-06-17T22:06:15|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\Users\lab\.p2\pool\plugins\org.python.pydev.core_6.4.4.201807281807\pysrc\pydevd.py", line 1709, in <module>
    main()
  File "C:\Users\lab\.p2\pool\plugins\org.python.pydev.core_6.4.4.201807281807\pysrc\pydevd.py", line 1703, in main
    globals = debugger.run(setup['file'], None, None, is_module)
  File "C:\Users\lab\.p2\pool\plugins\org.python.pydev.core_6.4.4.201807281807\pysrc\pydevd.py", line 1050, in run
    return self._exec(is_module, entry_point_fn, module_name, file, globals, locals)
  File "C:\Users\lab\.p2\pool\plugins\org.python.pydev.core_6.4.4.201807281807\pysrc\pydevd.py", line 1057, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "C:\Users\lab\.p2\pool\plugins\org.python.pydev.core_6.4.4.201807281807\pysrc\_pydev_imps\_pydev_execfile.py", line 25, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "C:\Users\lab\Documents\foundry_scope\ScopeFoundryHW\picoquant\hydraharp_test_app.py", line 25, in <module>
    app = HydraHarpTestApp()
  File "C:\Users\lab\Documents\foundry_scope\ScopeFoundry\base_app.py", line 340, in __init__
    self.setup_default_ui()
  File "C:\Users\lab\Documents\foundry_scope\ScopeFoundry\base_app.py", line 380, in setup_default_ui
    measure.setup_figure()
  File "C:\Users\lab\Documents\foundry_scope\ScopeFoundryHW\picoquant\hydraharp_optimizer.py", line 72, in setup_figure
    self.settings.SyncRate_visible.connect_to_widget(self.ui.SyncRate_checkBox)
AttributeError: 'QMainWindow' object has no attribute 'SyncRate_checkBox'

